### PR TITLE
Fix default .yaml configuration file access for dotnet-gcmon

### PR DESCRIPTION
**Pull request by chrisnas**
- look into the same folder as the running assembly to find the default .yaml configuration file
- refactor command line arguments error handling so no more exceptions are visible to the user (only message describing the issue)
- refactor session lifetime to avoid very unlikely race condition if the user exits too soon (before the session gets created)
 
image

### CODE REVIEW

File: src/dotnet-gcmon/readme.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+How to make a global .NET CLI tool to monitor garbage collection in real time?
```
Comment (MokoSan) [inline src/dotnet-gcmon/readme.txt]: Is this file redundant now that the [README.md](https://github.com/Maoni0/realmon#how-to-generate-the-global-net-cli-tool-dotnet-gcmon) contains the same information?

Comment (chrisnas) [reply src/dotnet-gcmon/readme.txt]: Good point: I thought I deleted it in a commit of the original PR.

**Comment by chrisnas** [comment]
I have also updated the nuget version to 0.2.0

File: src/dotnet-gcmon/readme.txt (hunk 1 above)
Comment (chrisnas) [reply src/dotnet-gcmon/readme.txt]: ✅

File: src/GCRealTimeMon/Program.cs [hunk 2]
```diff
@@ -1,16 +1,17 @@
 ﻿using System;
-using Microsoft.Diagnostics.Tracing;
-using Microsoft.Diagnostics.Tracing.Session;
-using Microsoft.Diagnostics.Tracing.Parsers;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
```
Comment (Maoni0) [inline src/GCRealTimeMon/Program.cs:5]: what are we using from Reflection? :)

**Review by Maoni0 — APPROVED** [review]
LGTM! thanks so much for fixing this :)

**Pull request merged**